project/SensibleSettings.scala

```diff
@@ -143,15 +151,15 @@ object Sensible {
     "com.google.code.findbugs" % "jsr305" % "3.0.1" % "provided"
   )
 
-  def testLibs(config: String = "test") = Seq(
+  def testLibs(config: String = "test") = Def.setting(Seq(
```

Comment:
I will probably remove a bunch of stuff from this `testLibs` when I publish it as a separate plugin